src/block_cache/block_chunkcache.c

```diff
@@ -20,6 +20,88 @@
 #define WT_CHUNK_OFFSET(chunkcache, offset) \
     (wt_off_t)(((size_t)offset / (chunkcache)->chunk_size) * (chunkcache)->chunk_size)
 
+/*
+ * __chunkcache_create_metadata_file --
+ *     Create the table that will persistently track what chunk cache content is on disk.
+ */
+static int
+__chunkcache_create_metadata_file(
+  WT_SESSION_IMPL *session, uint64_t capacity, unsigned int hashtable_size, size_t chunk_size)
+{
+    char cfg[128];
+    WT_RET(
+      __wt_snprintf(cfg, sizeof(cfg),
+        "app_metadata=\"version=1,capacity=%" PRIu64 ",buckets=%u,chunk_size=%" WT_SIZET_FMT "\","
+        "key_format=" WT_CC_KEY_FORMAT ",value_format=" WT_CC_VALUE_FORMAT,
+        capacity, hashtable_size, chunk_size));
+
+    return (__wt_session_create(session, WT_CC_URI, cfg));
+}
+
+/*
+ * __chunkcache_get_metadata_config --
+ *     If present, retrieve the on-disk configuration for the chunk cache metadata file. The caller
+ *     must only use *config if *found is true. The caller is responsible for freeing the memory
+ *     allocated into *config.
+ */
+static int
+__chunkcache_get_metadata_config(WT_SESSION_IMPL *session, bool *found, char **config)
+{
```

Comment:
Why not return WT_NOTFOUND if not found, rather than having a separate bool return variable?

Reply:
Good call - my original approach was quite different and this was left over. Changed in f332172.